x/gov/keeper/tally_test.go

```diff
@@ -410,7 +410,9 @@ func TestTallyJailedValidator(t *testing.T) {
 
 	_ = staking.EndBlocker(ctx, app.StakingKeeper)
 
-	app.StakingKeeper.Jail(ctx, sdk.ConsAddress(val2.GetConsPubKey().Address()))
+	consKey, err := val2.TmConsPubKey()
```

Comment:
why does this need to be a tendermint key? It seems this PR intertwined tendermint keys even deeper into the sdk..